src/viking.js

```diff
@@ -1,11 +1,102 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        this.damage = damage;
+        this.health -= damage
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength);
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength
+    }
+    battleCry() {
+        let odim = "Odin Owns You All!";
+        return odim;
+    }
+    receiveDamage(damage) {
+        this.damage = damage;
```

Comment:
Here, `damage` is going to be the argument of your `receiveDamage()` method, so, you don't need to reference it to the class as you are receiving some value (later on) and that's the value you are going to use to remove it from the `health`.